scripts/update-manpages

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
```

Comment:
Please add some metadata so that it will work with `find-tools`. `scripts/README.md` describes how to do that.